hub/powertoys/fancyzones.md

```diff
@@ -9,41 +9,41 @@ no-loc: [PowerToys, Windows, FancyZones, Fancy, Zone, Zones, Win]
 
 # FancyZones utility
 
-FancyZones is a window manager utility for arranging and snapping windows into efficient layouts to improve the speed of your workflow and restore layouts quickly. FancyZones allows you to define a set of zone locations to be used as targets for windows on your desktop. When you drag a window into a zone, or enter the associated keyboard shortcut, the window is resized and repositioned to fill that zone.
+FancyZones is a window manager utility for arranging and snapping windows into efficient layouts to improve your workflow and restore layouts quickly. FancyZones allows you to define a set of zone locations to be used as targets for windows on your desktop. When you drag a window into a zone, or enter the associated keyboard shortcut, the window is resized and repositioned to fill that zone.
 
-## Snapping to a single zone with the mouse
+## Snapping to a single zone with mouse
 
-First, drag the window. By default, you will also need to press and hold the <kbd>Shift</kbd> key (This setting can be disabled via Settings). You'll see the zones appear. As you move your mouse, hovering over a zone will highlight that zone. 
+First, drag the window. By default, you will also need to press and hold the <kbd>Shift</kbd> key (This setting can be disabled via Settings). You'll see the zones appear. As you move your mouse, hovering over a zone will highlight that zone.
```

Comment:
```suggestion
Drag the window. By default, you'll also need to select and hold the <kbd>Shift</kbd> key. You'll see the zones appear. As you move your mouse, hovering over a zone will highlight that zone.
```

I don't think you need to specify that the activation key can be disabled at this point because it is implied with 'By default'.

'Select' is preferred to 'press' when [describing keyboards](https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/term-collections/keys-keyboard-shortcuts).